tests/playbooks/tests_wireless_and_network_restart.yml

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Play for testing wifi and network restart
+  hosts: all
+  vars:
+    interface: wlan0
+    profile: "{{ interface }}"
+    package: NetworkManager-wifi
+    wifi_restart_network: false
+    network_connections_result_error:
+      NetworkManager needs to be restarted to be able to proceed
+      because wireless and team interfaces are defined. This might
+      disturb the connectivity of the managed system. Please set
+      `network_allow_restart` to `true` if you are prepared for this.
+      Notice that the necessary action is to install NetworkManager-wifi or
+      NetworkManager-team plugin and to restart NetworkManager.
+    lsr_fail_debug:
+      - __network_connections_result
+  tasks:
+    - name: Show playbook name
+      debug:
+        msg: "this is: playbooks/tests_wireless_and_network_restart.yml"
+      tags:
+        - always
+
+    - name: Cannot test ostree systems because the package can not be removed
+        or installed - 'NetworkManager-wifi'
```

Comment:
If we are using the jinja2 syntax for the '{{ package }}', the {{ package }}' should be appeared at the end of the task name.

Reply:
FWIW, `cannot` is more common than `can not`.

Reply:
Right, thanks for pointing it out, I will address it in another PR.